docs/sources/setup-grafana/configure-docker.md

```diff
@@ -8,45 +8,70 @@ keywords:
+Running Grafana on Docker may require some additional configuration which depends on the complexity of your environment. We will explain some of those e.g. using different images, changing logging levels, defining secrets on Cloud (e.g. AWS), configuring plugins etc.
 
 > **Note:** These examples use the Grafana Enterprise docker image. You can use the Grafana Open Source edition by changing the docker image to `grafana/grafana-oss`.
 
-## Save your Grafana data
+# Supported Docker Image variants
 
-If you do not designate a location for information storage, then all your Grafana data disappears as soon as you stop your container. To save your data, you need to set up persistent storage or bind mounts for your container.
+You can install and run Grafana using the following official Docker images.
 
-### Run Grafana container with persistent storage (recommended)
+**Grafana Enterprise**: `grafana/grafana-enterprise`
 
-```bash
-# create a persistent volume for your data in /var/lib/grafana (database and plugins)
-docker volume create grafana-storage
+**Grafana Open Source**: `grafana/grafana-oss`
 
-# start grafana
-docker run -d -p 3000:3000 --name=grafana -v grafana-storage:/var/lib/grafana grafana/grafana-enterprise
-```
+Each edition is available in two variants: Alpine and Ubuntu. See below.
+
+## Alpine image (recommended)
+
+The default images are based on the popular [Alpine Linux project](http://alpinelinux.org/), available in the [Alpine official image](https://hub.docker.com/_/alpine). Alpine Linux is much smaller than most distribution base images, and thus leads to slimmer and more secure images.
+The Alpine variant is highly recommended when security is important and you want an  image size that’s as small as possible. Note that the Alpine variant uses [musl libc](http://www.musl-libc.org/) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most software doesn’t have an issue with this, so this variant is usually a very safe choice.
+
+## Ubuntu image
+
+**Grafana Enterprise**: `grafana/grafana-enterprise:<version>-ubuntu`
 
-### Run Grafana container using bind mounts
+**Grafana Open Source**: `grafana/grafana-oss:<version>-ubuntu`
 
-You may want to run Grafana in Docker but use folders on your host for the database or configuration. When doing so, it becomes important to start the container with a user that is able to access and write to the folder you map into the container.
+These images are based on [Ubuntu](https://ubuntu.com/), available in the [Ubuntu official image](https://hub.docker.com/_/ubuntu). It is an alternative image for those who prefer an Ubuntu-based image and/or are dependent on certain tooling not available for Alpine.
```

Comment:
```suggestion
These images are based on [Ubuntu](https://ubuntu.com/) and can be accessed through the [Ubuntu official image](https://hub.docker.com/_/ubuntu). This is a good option for users who prefer an Ubuntu-based image or require specific tools unavailable on Alpine.
```

Reply:
These images is reffering to :
Grafana Enterprise: grafana/grafana-enterprise:<version>-ubuntu
and
Grafana Open Source: grafana/grafana-oss:<version>-ubuntu

Reply:
Got it. Modified my suggestion. Feel free to accept if it looks good to you.